tests_integration/__tests__/__snapshots__/early-exit.js.snap

```diff
@@ -311,6 +296,15 @@ exports[`throw error with --find-config-path + multiple files (stdout) 1`] = `""
 
 exports[`throw error with --find-config-path + multiple files (write) 1`] = `Array []`;
 
+exports[`throw error with --help not-found (stderr) 1`] = `
+"[error] Invalid --help value. Expected \\"arrow-parens\\", \\"bracket-spacing\\", \\"color\\", \\"config\\", \\"config-precedence\\", \\"cursor-offset\\", \\"editorconfig\\", \\"file-info\\", \\"find-config-path\\", \\"h\\", \\"help\\", \\"ignore-path\\", \\"insert-pragma\\", \\"jsx-bracket-same-line\\", \\"l\\", \\"list-different\\", \\"loglevel\\", \\"no-bracket-spacing\\", \\"no-color\\", \\"no-config\\", \\"no-editorconfig\\", \\"no-semi\\", \\"parser\\", \\"plugin\\", \\"plugin-search-dir\\", \\"print-width\\", \\"prose-wrap\\", \\"range-end\\", \\"range-start\\", \\"require-pragma\\", \\"semi\\", \\"single-quote\\", \\"stdin\\", \\"stdin-filepath\\", \\"support-info\\", \\"tab-width\\", \\"trailing-comma\\", \\"use-tabs\\", \\"v\\", \\"version\\", \\"with-node-modules\\" or \\"write\\", but received \\"not-found\\".
```

Comment:
Would it be easy to make this into a bulleted list? I feel like it would be easier to read that way.

Reply:
Ref: ikatyang/vnopts#9

Something like this?

![image](https://user-images.githubusercontent.com/8341033/45137126-e30b7a80-b1d9-11e8-948e-de59d7bd1be9.png)

Reply:
That looks great!

Reply:
I personally wouldn't show every option available.. in this case, I'd just show "Invalid --help value"

Reply:
Perhaps “Invalid --help value, expected an option name but got X.”?

Reply:
Since we called it `flag` in the usage, "expected `a flag`" should be better I think?

https://github.com/prettier/prettier/blob/4d488e8b81276e310b00c097b1ac9e385181c3cd/tests_integration/__tests__/__snapshots__/early-exit.js.snap#L124-L125